lib/vmpooler.rb

```diff
@@ -133,8 +133,17 @@ def self.config(filepath = 'vmpooler.yaml')
       parsed_config[:pools] = load_pools_from_redis(redis)
     end
 
+    # Marshal.dump is paired with Marshal.load to create a copy that has its own memory space
+    # so that each can be edited independently
+    # rubocop:disable Security/MarshalLoad
+
+    # retain a copy of the pools that were observed at startup
+    serialized_pools = Marshal.dump(parsed_config[:pools])
```

Comment:
could do one liner here
```suggestion
    parsed_config[:pools_at_startup] = Marshal.load(Marshal.dump(parsed_config[:pools]))
```

Reply:
It think the one liner is a little hard to follow, personally